src/components/table/partial-styles/tabulator-custom-styles.scss

```diff
@@ -1,32 +1,87 @@
 @use '../../../style/mixins';
 @use '../../../style/functions';
 
+.interactive-feedback {
+    // This is a div, injected by us into all rows.
+    // We use it to visualize interactive feedback.
+    position: absolute;
+    inset: 0;
+    min-height: 0.125rem;
+
+    // We could use `@include mixins.is-flat-clickable()`,
+    // But the "arguments" of the mixin would make the CSS for
+    // even and odd row colors and background color very complicated.
+    // So instead, I've copy/pasted the useful parts of the mixin here
+    transition: color 0.2s ease, background-color 0.2s ease,
+        box-shadow 0.2s ease, transform 0.1s ease-out;
+}
+
+.tabulator-row:not(.tabulator-calcs) {
+    .tabulator-cell {
+        // We don't want the cells to block
+        // the hovering possibility of `interactive-feedback` div.
+        pointer-events: none;
+        // but we want the content in them
+        // (e.g. links, phone numbers, chips, components)
+        // to remain interactive.
+        > * {
+            pointer-events: all;
+        }
+    }
+
+    &.tabulator-selectable {
+        &:hover {
+            z-index: $table--has-interactive-rows--selectable-row--hover;
+        }
+
+        &.active {
+            .interactive-feedback {
+                background-color: rgb(
+                    var(--table-row-background-color--active)
+                );
+                box-shadow: var(--button-shadow-inset-pressed);
+            }
+
+            &:before {
+                $width-of-selected-row-indicator: 0.2rem;
+                content: '';
+                display: inline-block;
+                box-sizing: border-box;
+                position: sticky;
+                z-index: $table--has-interactive-rows--selectable-row--hover + 1;
+                inset: 0 0 auto 0.25rem;
```

Comment:
But the inset used to be 0 0 auto 0, just curious why it changed 😅

Reply:
Oh I was hoping the reviewer will miss this 😄 

I was too lazy and didn't really want to make a new commit for this change.

I just felt that the green circle was too close to the edge.
<img width="122" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/94045da3-2f36-476d-aeaa-65675e17e58c">
But I'll take it out from this commit now.